src/graphql/fragments/loanCardFields.graphql

```diff
@@ -1,5 +1,6 @@
 fragment loanCardFields on LoanBasic {
 	id
+	distributionModel
```

Comment:
These additions here are the ones that were used in the loan card but were not present in this fragment